### PR TITLE
fix: allow display name change for verified when empty

### PR DESCRIPTION
**Pull request by hirbod**
# Why

Verified users can't change name once they are verified. We have edge cases, where we have verified users but they still haven't entered a display name.

https://showtime-rq88331.slack.com/archives/C02PXGK3V8D/p1680502899279759

# How
Allow change of the name (not username) if the field is empty. @jorgeavaldez can you take care of the backend part please? Thanks King!

# Test Plan

### CODE REVIEW

File: packages/app/components/edit-profile.tsx [hunk 1]
```diff
@@ -135,24 +134,10 @@ export const EditProfile = () => {
...
-        const foundLink = user.data.profile.links.find(
-          (l) => l.type_id === s.id
-        );
-
-        if (foundLink) {
-          links[s.id] = foundLink.user_input;
-        }
-      });
-    }
-
```
Comment (hirbod) [inline packages/app/components/edit-profile.tsx:150]: I removed these since we're not using this fields anymore

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️ (post BE changes)

**Review by alantoa — APPROVED** [review]
LGTM

**Comment by hirbod** [comment]
Backend was merged, we can merge as well now.

**Pull request merged**